docs/QuickStartGuides/1-GettingRunning.md

```diff
@@ -0,0 +1,52 @@
+# Hello World
+
+Symbulation allows you to compile your C++ to target
+
+- running at the command line (e.g., native machine code) and
+- running in the web browser (e.g., javascript assembly code).
+
+This how-to aims to walk you through all the nitty gritty required to
+successfully compile a simple "Hello World" for both targets.
```

Comment:
Instead of "Hello World", perhaps "default" since it isn't a library (yet) and so doesn't have a "Hello World". 

"successfully compile the default world of hosts and symbionts for both targets."